### ISSUE TITLE
TypeError: (void 0) is not a function

### ISSUE THREAD

**Issue by bmstefanski** [emilkowalski/vaul#120]
Hi! Thanks for the awesome library!

I encountered a bug with 0.6.6 version that causes `next build` to throw `TypeError: (void 0) is not a function`. Going back to 0.6.5 fixes the issue.

My node.js version: 18.16.0

```
TypeError: (void 0) is not a function
    at Je (/home/ubuntu/apps/xd/releases/xd/node_modules/vaul/dist/index.js:3:9113)
```

**Comment by Alexandredc** [comment]
Same problem here !

**Comment by emilkowalski** [comment]
Released 0.6.7 which fixes the issue. Will add a changelog soon. Thank you for reporting!

**Comment by gablabelle** [comment]
> Released 0.6.7 which fixes the issue. Will add a changelog soon. Thank you for reporting!

It did fixed the issue, thanks 👍
